contract/src/sanity.rs

```diff
@@ -83,6 +83,7 @@ pub fn sanity(store: &dyn Storage, env: &Env) {
             assert_eq!(tokens.0, pool_size.0);
         }
 
+        // FIXME modify check to include in all wallets if they have a liquidity share
```

Comment:
Not sure if you want to address this as part of this PR or later.

Reply:
Fixed in the next PR.